modules/bibconvert/lib/bibconvert_xslt_engine.py

```diff
@@ -66,6 +67,10 @@
 try:
     # lxml
     from lxml import etree
+    lxml_version = etree.LXML_VERSION
+    if lxml_version < (2, 0, 11, 0):
+        raise ImportError("Minimum lxml version supported is 2.0.11.0, available is %s" % \
+                          ('.'.join([str(v) for v in lxml_version]),))
```

Comment:
@kaplun @kasioumis Any progress with moving requirement check?  Shouldn't we close #2275 since we started to discuss here?